Example/Example/CharactersViewController.swift

```diff
@@ -24,4 +24,13 @@ class CharactersViewController: ExampleViewController, BothamTableViewController
         pullToRefreshHandler.addTo(tableView)
         super.viewDidLoad()
     }
+    
+    override func viewDidAppear(animated: Bool) {
+        super.viewDidAppear(animated)
+        pullToRefreshHandler.beginRefreshing()
+        let delayTime = dispatch_time(DISPATCH_TIME_NOW, Int64(3 * Double(NSEC_PER_SEC)))
+        dispatch_after(delayTime, dispatch_get_main_queue()) {
+            self.pullToRefreshHandler.beginRefreshing()
+        }
```

Comment:
I don't understand this change. Can you explain us why you introduce a delay to call ``pullToRefreshHandler.beginRefreshing()`` again? And why do you call ``beginRefreshing`` during the ``viewDidApplear`` execution?